apps/meteor/tests/unit/app/importer/server/userConverter.spec.ts

```diff
@@ -120,12 +121,14 @@ describe('User Converter', () => {
 			},
 			...expectedData,
 		});
-
 		const converter = new UserConverter({ workInMemory: true });
-		sinon.stub(converter, 'generateTempPassword');
-		sinon.stub(converter, 'hashPassword');
-		converter.generateTempPassword.returns('tempPassword');
-		converter.hashPassword.callsFake((pass: string) => `hashed=${pass}`);
+
+		before(() => {
+			sinon.stub(converter, 'generateTempPassword');
+			sinon.stub(converter, 'hashPassword');
+			converter.generateTempPassword.returns('tempPassword');
+			converter.hashPassword.callsFake((pass: string) => `hashed=${pass}`);
+		});
```

Comment:
Is this related to the PR? 🤔 If not, please remove it

Reply:
Unit tests were failing here, so I had to add this to make sure the new unit tests were ok in this PR
@pierre-lehnen-rc will apply this fix in another PR (while this fix isn't merged, unit tests will fail) , so I'll remove this from here